src/js/gi/components/profile/CautionaryInformation.jsx

```diff
@@ -1,43 +1,52 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import { showModal } from '../../actions';
+
 import AlertBox from '../../../common/components/AlertBox';
 
 const TableRow = ({ description, thisCampus, allCampuses }) => {
+  if (!thisCampus && !allCampuses) return null;
+  const bold = description === 'Total Complaints';
   return (
     <tr>
-      <th><strong>{description}</strong></th>
-      <td className="number">{thisCampus}</td>
-      <td className="number">{allCampuses}</td>
+      <th><b>{description}</b></th>
```

Comment:
Is there a reason you're switching from the `<strong>` tag to `<b>`? Generally I prefer using `<strong>` so that it conveys mean to screen readers.